src/components/ReanimatedSwipeable.tsx

```diff
@@ -17,7 +16,7 @@ import {
   GestureUpdateEvent,
   PanGestureHandlerEventPayload,
   PanGestureHandlerProps,
-} from 'react-native-gesture-handler';
+} from '../.';
```

Comment:
Similarly to the other PR, I don't think it should be imported from `index`. While in this case, it doesn't introduce any cycles (this component is not imported in `index`), it may turn out to be problematic in the future if someone forgets about it.